tests/src/test/java/alluxio/client/FileOutStreamAsyncWriteIntegrationTest.java

```diff
@@ -69,6 +72,7 @@ public void asyncWriteEmptyFileTest() throws Exception {
     status = mFileSystem.getStatus(filePath);
     Assert.assertEquals(PersistenceState.PERSISTED.toString(), status.getPersistenceState());
 
-    checkWrite(filePath, mWriteAsync.getUnderStorageType(), 0, 0);
+    checkWrite(filePath, CreateFileOptions.defaults().setWriteType(WriteType.ASYNC_THROUGH).getUnderStorageType(),
```

Comment:
`CreateFileOptions.defaults().setWriteType(WriteType.ASYNC_THROUGH).getUnderStorageType()` is equivalent to `UnderStorageType.ASYNC_PERSIST`